include/ccl_utils.h

```diff
@@ -9,6 +9,11 @@
 
 CCL_BEGIN_DECLS
 
+typedef enum ccl_integration_t {
+  ccl_integration_quad = 500,  // GSL's quad
```

Comment:
So this method is the same as it was previously. Maybe we should call this `ccl_integration_qag_cquad` or something? After all, `c_quad` is the backuo method.

Reply:
good point (I named it from memory). Done.